conjure-java-jackson-serialization/src/main/metrics/jackson-registry-metrics.yml

```diff
@@ -1,12 +0,0 @@
-options:
-  javaPackage: com.palantir.conjure.java.serialization
-  javaVisibility: packagePrivate
-namespaces:
-  json.parser:
-    docs: Metrics produced instrumented Jackson components.
-    metrics:
-      string.length:
-        type: histogram
-        tags:
-          - format
-        docs: Histogram describing the length of strings parsed from input.
```

Comment:
This was a rename to `jackson-metrics` since `-registry` didn't make a lot of sense in the filename.